roles/ceph-rgw/tasks/common.yml

```diff
@@ -41,3 +41,13 @@
   when:
     - cephx | bool
     - item.item.copy_key | bool
+
+- name: copy SSL certificate & key data to certificate path
+  copy:
+    content: "{{ radosgw_frontend_ssl_certificate_data }}"
+    dest: "{{ radosgw_frontend_ssl_certificate }}"
+    owner: ceph
+    group: ceph
```

Comment:
This won't work for containerized deployment.

You need something like https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-rgw/tasks/common.yml#L6-L7